### PR TITLE
Translate Reference Guides

### PR DESCRIPTION
**Pull request by zweimach**
Please review my translation. Thank you.

### CODE REVIEW

File: docs/docs/guides.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Reference Guides
+title: Panduan Referensi
 overview: true
 ---
 
-Dive deeper into different topics around building with Gatsby, like sourcing data, building and styling pages, deployment, and more. While the [tutorials](/tutorial/) are step-by-step instructions, reference guides are resources about the various Gatsby development techniques.
+Gali lebih dalam tentang topik-topik seputar mengembangkan dengan Gatsby, seperti *sourcing* data, membuat dan menghias laman, *deployment*, dan banyak lagi. Meskipun [tutorial](/tutorial/) adalah petunjuk langkah demi langkah, panduan referensi adalah sumber daya pengetahuan tentang berbagai macam teknik pengembangan dengan Gatsby.
```
Comment (coroo) [inline docs/docs/guides.md]: Please help to change laman with halaman

**Review by coroo — CHANGES_REQUESTED** [review]
Thank you, please help us to update some comment abive

**Comment by zweimach** [comment]
@coroo, thank you for your feedback.

**Review by coroo — APPROVED** [review]
Hi @zweimach , thank you for your help.
Please dont forget to get your swag

**Pull request merged**